cmd/manager/main.go

```diff
@@ -38,6 +38,7 @@ var (
 func init() {
 	_ = scheme.AddToScheme(myscheme)
 	_ = v1alpha1.AddToScheme(myscheme)
+	_ = v1alpha1.AddToScheme(myscheme)
```

Comment:
was this autogenerated? they should already be in the same scheme. Probably a bug in kubebuilder, but you can delete this line

Reply:
OK